api/src/pcapi/routes/backoffice/templates/offerer/get.html

```diff
@@ -171,6 +173,51 @@ <h5 class="modal-title">Supprimer la structure {{ offerer.name }}</h5>
                    target="_blank"
                    class="fw-bold link-primary">Offres associées</a>
               </p>
+              <div>
+                {% if zendesk_sell_synchronisation_form %}
+                  <button class="btn btn-outline-primary btn-sm"
+                          data-bs-toggle="modal"
+                          data-bs-target=".pc-sync-zsell-modal"
+                          type="button">
+                    <i class="bi bi-arrow-repeat"></i>
+                    Zendesk Sell
+                  </button>
+                  <div class="modal modal-lg fade pc-sync-zsell-modal"
+                       tabindex="-1"
+                       aria-describedby="{{ sync_zsell_aria_described_by_id }}"
+                       aria-hidden="true">
+                    <div class="modal-dialog modal-dialog-centered">
+                      <div class="modal-content">
+                        <form action="{{ url_for('backoffice_web.zendesk_sell.update_offerer', offerer_id=offerer.id) }}"
+                              name="{{ url_for('backoffice_web.zendesk_sell.update_offerer', offerer_id=offerer.id) | action_to_name }}"
+                              method="post"
+                              data-turbo="false">
+                          {{ zendesk_sell_synchronisation_form.csrf_token }}
+                          <div class="modal-header"
+                               id="{{ sync_zsell_aria_described_by_id }}">
+                            <h5 class="modal-title">Synchroniser la structure {{ offerer.name }} sur zendesk sell</h5>
+                          </div>
+                          <div class="modal-body row">
+                            <div class="">
+                              <i class="bi bi-exclamation-diamond-fill text-warning align-middle me-3 ms-2"
+                                 style="font-size: 3.1rem"></i>
+                              <span class="lead">Vérifier que l’id structure/lieu est bien renseigné sur Zendesk Sell !</span>
```

Comment:
A priori pas besoin de l'id lieu pour la synchro structure, si ?
```suggestion
                              <span class="lead">Vérifier que l’id structure est bien renseigné sur Zendesk Sell !</span>
```